cluster/tla/consensus.tla

```diff
@@ -113,17 +110,10 @@ HandleJoinRequest(n, m, rcvrs) ==
   /\ m.term = currentTerm[n]
   /\ \/ /\ m.slot < firstUncommittedSlot[n]
      \/ /\ m.slot = firstUncommittedSlot[n]
-        /\ \/ m.laTerm = Nil
-           \/ m.laTerm = LastAcceptedTermInSlot(n)
-           \/ /\ m.laTerm /= Nil
-              /\ LastAcceptedTermInSlot(n) /= Nil
-              /\ m.laTerm <= LastAcceptedTermInSlot(n)
-              /\ electionValueForced[n]
+        /\ (m.laTerm /= Nil => LastAcceptedTermInSlot(n) /= Nil /\ m.laTerm <= LastAcceptedTermInSlot(n))
   /\ joinVotes' = [joinVotes EXCEPT ![n] = @ \cup { m.source }]
-  /\ electionValueForced' = [electionValueForced EXCEPT ![n] = 
-                               @ \/ (m.laTerm /= Nil /\ m.slot = firstUncommittedSlot[n])]
   /\ electionWon' = [electionWon EXCEPT ![n] = IsQuorum(joinVotes'[n], currentConfiguration[n])]
-  /\ IF electionValueForced'[n] /\ electionWon'[n] /\ \lnot electionWon[n]
+  /\ IF publishPermitted[n] /\ LastAcceptedTermInSlot(n) /= Nil /\ electionWon'[n] /\ \lnot electionWon[n]
```

Comment:
Is the `¬electionWon[n]` needed here, given that `publishPermitted` is checked?

Reply:
right, it's not needed => I've removed it in d95800d